mem/table.go

```diff
@@ -2,86 +2,279 @@ package mem
 
 import (
 	"bytes"
-	"encoding/binary"
+	"encoding/gob"
 	"fmt"
 	"io"
+	"strconv"
 
-	errors "gopkg.in/src-d/go-errors.v1"
+	errors "gopkg.in/src-d/go-errors.v0"
```

Comment:
why did you change to previous go-errors version?

Reply:
that was probably go-imports' thing

Reply:
It was indeed